print/templates/pattern_only.html

```diff
@@ -0,0 +1,25 @@
+    <main class="page pattern-onlyview" id="pattern-display">
+      <figure class="inksimulation pattern-only operator"
+              data-field-name="pattern-onlyview"
```

Comment:
Change the data-fiel-name to `pattern-onlyview-transform`
This will store the transform value if you reopen the print pdf file again